src/components/Login/Login.tsx

```diff
@@ -6,19 +6,28 @@ import { validateLoginData } from 'utils/Validations';
 
 import './Login.scss';
 import '../common_style/Form.scss'
+import { LoginError } from 'interfaces/LoginError';
 
 const Login = () => {
 	const [user, setUser] = useState({ email: '', password: '' })
-	const [errors, setErrors] = useState({ email: '', password: '', isError: false })
+	const [errors, setErrors] = useState<LoginError>()
 
 	const handleChange = (e: any) => {
 		setUser(Object.assign({}, user, { [e.target.name]: e.target.value }))
 	}
 
 	const handleSubmit = (e: any) => {
 		e.preventDefault();
-		
-		if (validate()) {
+
+		setErrors(validateLoginData(user))
+		setTimeout(
+			() => handleLogin(), 
+			300
+		);
+	}
+
+	const handleLogin = () => {
+		if (errors?.isError) {
```

Comment:
A better way to handle this would be to initially act as if there are errors and disable the submit button in response to that. As the user is typing, you can check whether or not there was a pause and validate what was entered into the field. Once all fields are validated, the submit button can be enabled and allow you to submit the form.